libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/Microsoft.Bot.Builder.Integration.AspNet.Core.csproj

```diff
@@ -24,12 +24,14 @@
 	<ItemGroup>
 		<PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="all" />
 		<PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.1" />
+		<PackageReference Include="Microsoft.Bot.Protocol" Version="1.0.0" />
+		<PackageReference Include="Microsoft.Bot.Protocol.WebSockets" Version="1.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
     <PackageReference Include="Microsoft.Bot.Builder" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
     <PackageReference Include="Microsoft.Bot.Builder" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
-		<PackageReference Include="Microsoft.Net.Http.Headers" Version="2.0.1" />
-		<PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+		<PackageReference Include="Microsoft.Net.Http.Headers" Version="2.2.0" />

```

Comment:
These package updates were brought on by the inclusion of the Protocol and Protocol.WebSockets packages. They don't seem to cause any conflicts with the rest of Bot Builder, but if anyone has concerns please let me know!